packages/strapi-generate-api/templates/bookshelf/service.template

```diff
@@ -29,22 +29,19 @@ module.exports = {
       .map(ast => ast.alias);
 
     return <%= globalID %>.query(function(qb) {
-      _.forEach(filters.where, (where, key) => {
```

Comment:
@Aurelsicoko @lauriejim We need to find a way to let the users modify the logic if they want, but the question is: "Do they really need to modify the Core Query System?"

Reply:
I mean we can, for instance, put this utility functions inside of a folder that they can update but I don't see why should we duplicate the code in the different places (API Services / Content Manager and so on)

Reply:
You're right! We could centralise everything in a single place but the users need to keep control if they want to edit something. It should not be hidden somewhere to avoid updates...

Reply:
@Aurelsicoko I think doing something like what @soupette did with the docs plugin and centralize everything but allow for model level overrides